### PR TITLE
NEXT: Descriptions and Cleanup

### PR DESCRIPTION
**Pull request by DeathlyPlays**
- Magma Armor in the code says it changes to Battle Armor not Shell Armor

### CODE REVIEW

**Comment by DeathlyPlays** [comment]
Didn't mean to add the Gen Next work :v

**Comment by DeathlyPlays** [comment]
just the README but oh well

**Comment by panpawn** [comment]
You could just rebase this to make the changes what you had intended.

**Comment by DeathlyPlays** [comment]
At this point fuck it >_>

**Comment by panpawn** [comment]
Without rebasing, it makes your actual changes become rather hard to follow.

That, combined with using unhelpful commit names such as "UPDATES" and "Oops" makes for a more annoying proposal to review.

Rebasing isn't difficult, and if you need help with rebasing feel free to ask me or the Dev room. Knowing how to rebase is really useful and it makes for a much easier way of communicating what you actually want to change.

**Comment by DeathlyPlays** [comment]
@Zarel done and ready for you to review the descriptions

**Comment by Zarel** [comment]
Did you just ignore the part where panpawn told you to rebase?

**Comment by DeathlyPlays** [comment]
I will have to learn how to rebase o3o

**Comment by Zarel** [comment]
`git rebase -i HEAD~6`

will give you an interactive rebase of the last 6 commits, at which point you can edit the text to rearrange, merge (squash), and rename those commits.

Tell me if you need more help.

**Comment by DeathlyPlays** [comment]
Ok will do when I get computer access, thanks
